unit10/unit10.js

```diff
@@ -0,0 +1,145 @@
+//Bai : 1  in một bảng của một số bất kì với các số từ 1 đến 10 và hiển thị ra kết quả 
+const tableMultiplication = (number) => {
+  if(number >= 1 && number <= 10) {
+    console.log('In bảng nhân:');
+    for(let i = 1 ; i <= 10 ; i ++){
+      console.log(number + ' x ' + i + ' = ' + number * i);
+    }
+  }else {
+    console.log('mời bạn nhập lại môt số trong khoảng từ 1 đến 10');
+  }
+}
+
+// Bài 2 : hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 in các số chẵn từ 1 đến n 
+const InEvenNumber = (number) => {
+  if(number >= 1 && number <= 30) {
+    for(let i = 1 ; i <= number ; i ++){
+      if(i % 2 == 0 ){
+        console.log(i);
+      }
+    }
+  }else {
+    console.log('mời bạn nhập lại môt số trong khoảng từ 1 đến 30');
+  }
+}
+
+//Bài 3 : Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 Tính tổng từ 1 đến n
+const sum = (number) => {
+  if(number >= 1 && number <= 30) {
+    let sum = 0 ; 
+    for(let i = 1 ; i <= number ; i ++){
+      sum = sum + i
+    }
+    console.log('tổng các số từ 1 đến ' + number + ' là : ' + sum)
+  }else {
```

Comment:
Đưa else lên đầu như các comments bài trên

```suggestion
  } else {
```